tweets/OpenJSWorld.tweet

```diff
@@ -0,0 +1,4 @@
+The @openjsf is holding their free event, #OpenJSWorld, on June 2nd. There'll be number of excellent talks, including a number on Node.js.
```

Comment:
The wording could be a little better. Here's a stab at simply updating it a little.

```suggestion
The @openjsf is holding their free event, #OpenJSWorld, on June 2nd. There'll be a number of excellent talks, including a several on Node.js.
```

Reply:
New wording looks good, i'd just delete the "a" before several.